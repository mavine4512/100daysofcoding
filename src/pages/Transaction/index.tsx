import React from 'react';

const Transaction = (): JSX.Element => {
    return <div><h5>Welcome to Transaction</h5></div>;
}

export default Transaction;
