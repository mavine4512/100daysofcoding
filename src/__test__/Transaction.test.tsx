import React from 'react';
import ReactDOM from 'react-dom';
import Transaction from "../pages/Transaction";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom'

it('renders correctly Transaction', ()=>{
    const  div =document.createElement('h5');
    ReactDOM.render(<Transaction/>, div);
    ReactDOM.unmountComponentAtNode(div)
    expect(div).toMatchSnapshot();

})

it('renders a message', ()=>{
    const {getByText,unmount} = render(<Transaction/>)
    expect(getByText('Welcome to Transaction')).toBeInTheDocument()

    unmount()
})
