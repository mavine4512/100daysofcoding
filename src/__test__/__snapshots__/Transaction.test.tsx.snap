// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `[Function]`;

exports[`renders correctly Transaction 1`] = `<h5 />`;
